dockstore-webservice/src/main/java/io/dockstore/webservice/api/PrivilegeRequest.java

```diff
@@ -0,0 +1,32 @@
+package io.dockstore.webservice.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+
+/**
+ * This is an object to encapsulate the privilege request. Used for requests in
+ * /users/user/{userid}/privileges
```

Comment:
make this endpoint /users/{userId}/privledges to be more restful

Reply:
Done.